seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1137,7 +1137,7 @@
         "  ",
         "  ",
         "</style>",
-        "<a href=\"https://atjonathan.github.io/catphotoapp\" target=\"_blank\">CatPhotoApp</a>"
+        "<a href=\"http://freecatphotoapp.com/" target=\"_blank\">CatPhotoApp</a>"
```

Comment:
The endquote needs escaped like
` "<a href=\"http://freecatphotoapp.com/\" target=\"_blank\">CatPhotoApp</a>" `